conbench/api/compare.py

```diff
@@ -403,7 +400,7 @@ def get(self, compare_ids) -> f.Response:
 
             This endpoint will return a list of comparison objects, pairing benchmark
             results from the given baseline and contender runs that have the same case,
-            context, hardware, repository, and unit. The comparison object is the same
```

Comment:
Thank you for updating the spec here. This makes it visible for us how far-reaching a patch like this is, and a diff like this makes it much easier to go back to this topic in the future.